tests/test_int_conformance.py

```diff
@@ -19,6 +19,13 @@
 
 @pytest.mark.parametrize("in_type,pd_old_type,pd_new_type", parameters)
 def test_int_csv(in_type, pd_old_type, pd_new_type):
+    """Testing csv mapping from pyarrow to Pandas data types.
+
+    Args:
+        in_type (str): pyarrow data type read in from the csv.
+        pd_old_type (str): old pandas data type mapping.
+        pd_new_type (str): new pandas data type mapping, set as default.
```

Comment:
I'm not sure what 'set as default' refers to in this line and the change below. Won't this test check both the old mapping and the new mapping? And I think it requires an expected value for both.